components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -69,16 +69,26 @@ const ContributorProfileInfo = ({
                 "I am an open source developer with a passion for music and video games. I strive to improve the open source community and am always looking for new ways to contribute."}
             </p>
             <div className="flex flex-col text-sm mt-2 text-light-slate-9 gap-2">
-              {displayLocalTime && <span className="flex gap-2 items-center">
-                <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
-              </span>}
+              {displayLocalTime && (
+                <span className="flex gap-2 items-center">
+                  <FiClock className="text-light-slate-9" /> Local time:
+                  <span> {getTimeByTimezone(timezone ? getTimezone(timezone) : 1)}</span>
```

Comment:
```suggestion
                  <span> {getTimeByTimezone(timezone ? getTimezone(timezone) : 1)}</span>
```

Why `: 1`

Reply:
nvm, I looked at the code 😅. I think its better than nothing, but we probably want to UTC with no offset be the default or show nothing at all. But since is scope creep, i'd rather merge and move on.